source/configuration/read.txt

```diff
@@ -234,8 +219,8 @@ Partitioner Configuration
 ``ShardedPartitioner`` Configuration
 ````````````````````````````````````
 
-The ``ShardedPartitioner`` automatically determines 
-partitions to use based on your shard configuration. 
+The ``ShardedPartitioner`` automatically determines
+partitions to use based on your shard configuration.
 
 This partitioner is not compatible with hashed shard keys.
```

Comment:
I'm guessing you meant the line that reads "This partitioner is not compatible with hashed shard keys" rather than the description "The ShardedPartitioner automatically determines the partitions...".